windows/image.cpp

```diff
@@ -0,0 +1,31 @@
+#include "uipriv_windows.hpp"
+// stubbed out windows image list implementation.
+// Required for uiTable control, but windows implemenation
+// doesn't currently have image support.
+
+struct uiImage {
+	double width;
+	double height;
+	// HIMAGELIST images;

```

Comment:
Not a review comment, just a bit of information: I don't think we'll be able to use HIMAGELIST here, since that deals with a set of images that have the same pixel sizes, while uiImage deals with a set of images that has different pixel sizes but the same physical size (for DPI independence). Don't worry about changing anything; I'll work on it later if you don't already have another PR with it.

Reply:
Yep, I agree. Wasn't planning to go anywhere further with HIMAGELIST, Just stubbing out enough to get the tables compiling. and threw in the comment while I was in listview mode. I didn't spot the multi-size vs different-images distinction.